Microsoft.Toolkit.Uwp.SampleApp/SamplePages/ColorPickerButton/ColorPickerButtonXaml.bind

```diff
@@ -0,0 +1,122 @@
+﻿<Page
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:controls="using:Microsoft.Toolkit.Uwp.UI.Controls"
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    mc:Ignorable="d">
+
+  <Page.Resources>
+    <SolidColorBrush Color="{ThemeResource SystemChromeLowColor}" x:Key="SystemControlForegroundChromeLowBrush"/>
+  </Page.Resources>
+
+  <Grid>
```

Comment:
Like the different sample layouts here, but seeing the drop-down buttons pushed pretty low in the content area:

![image](https://user-images.githubusercontent.com/24302614/88101706-3d562000-cb53-11ea-8a66-5b70350ed506.png)

This doesn't work well if the window is smaller. Should we just have these all stacked on one another in a list that's more scrollable or something?

Reply:
I never really liked the layout here either but couldn't think of a better idea and didn't want to spend much time on it until there was feedback. It's a good idea to put the descriptions closer to the button. I can just throw the border and ColorPickerButton into a StackPanel and center them both. I think a list would not take enough advantage of horizontal space.

Reply:
I put the description next to the ColorPickerButton now. It's still not perfect with small screen sizes (descriptions don't resize) but I think it's good enough. If someone comes up with a bright idea to make it all look prettier it would be welcome.